android/java/org/chromium/chrome/browser/crypto_wallet/activities/BuySendSwapActivity.java

```diff
@@ -1189,14 +1252,16 @@ public FilterTextWatcherSendToAddr() {
             mValidator = new Validations.SendToAccountAddress();
         }
 
+        void setCoinType() {}
```

Comment:
why do we need that?

Reply:
I was thinking of an another solution before implemented a better one, don't need, removed.